src/autolabel/tasks/attribute_extraction.py

```diff
@@ -33,7 +33,7 @@
 class AttributeExtractionTask(BaseTask):
     NULL_LABEL = {}
     DEFAULT_TASK_GUIDELINES = "You are an expert at extracting attributes from text. Given a piece of text, extract the required attributes."
-    DEFAULT_OUTPUT_GUIDELINES = "You will return the extracted attributes as a json with the following keys:\n{attribute_json}"
+    DEFAULT_OUTPUT_GUIDELINES = "You will return the extracted attributes as a json with the following keys:\n{attribute_json}. Only output labels for the fields that exist in the provided prompt. Do not output anything for attributes that are irrelevant to the given input."
```

Comment:
let's be consistent in instructing the LLM with fields vs attributes and prompt vs input ? 

Something like only include attributes in the JSON that have a valid extracted value for the given input ?

Reply:
Makes sense, have pushed a fix for this!